AY2021S2-CS2103-T14-1/tp
# Fix #202: Update product image

## What this does
Fixes #202 by updating product image.

## How to test
1. cd to `docs/` folder of the project.
2. Run `bundle exec jekyll serve`.
3. Open the jekyll-hosted website in your browser (default address: http://127.0.0.1:4000).
4. Check that the landing page has the updated image.
4. Check that the UG has the updated image in the Quick Start section
